src/neo/Wallets/SQLite/UserWallet.cs

```diff
@@ -376,5 +376,25 @@ public override bool VerifyPassword(string password)
         {
             return password.ToAesKey().Concat(salt).ToArray().Sha256().SequenceEqual(LoadStoredData("PasswordHash"));
         }
+
+        private static byte[] Encrypt(byte[] data, byte[] key, byte[] iv)
+        {
+            if (data == null || key == null || iv == null) throw new ArgumentNullException();
+            if (data.Length % 16 != 0 || key.Length != 32 || iv.Length != 16) throw new ArgumentException();
+            using Aes aes = Aes.Create();
+            aes.Padding = PaddingMode.None;
+            using ICryptoTransform encryptor = aes.CreateEncryptor(key, iv);
+            return encryptor.TransformFinalBlock(data, 0, data.Length);
+        }
+
+        private static byte[] Decrypt(byte[] data, byte[] key, byte[] iv)
+        {
+            if (data == null || key == null || iv == null) throw new ArgumentNullException();
+            if (data.Length % 16 != 0 || key.Length != 32 || iv.Length != 16) throw new ArgumentException();
+            using Aes aes = Aes.Create();
+            aes.Padding = PaddingMode.None;
```

Comment:
Can we specify the mode?
```suggestion
            aes.Padding = PaddingMode.None;
            aes.Mode = CipherMode.ECB;
```

Reply:
I think we don't change the code to keep compatibility.

Reply:
> CBC itself can also be considered vulnerable in certain situations, specifically the use of predictable IVs and unauthenticated decryption can allow you to guess plaintexts.

https://crypto.stackexchange.com/questions/225/should-i-use-ecb-or-cbc-encryption-mode-for-my-block-cipher

Reply:
I think that we are not vulnerable because we don't crypt multiple messages, but I think that ECB it safer.

Reply:
In this case, @shargon, the current used here is `aes.Mode = CipherMode.CBC;`, because of the initialization vector `iv`;

I think it is good to make this clear, @erikzhang, that the generic `Aes.Create()` expects `CBC`.